tests/Microsoft.Identity.Test.Common/Core/Mocks/MockHelpers.cs

```diff
@@ -341,7 +341,7 @@ public static HttpResponseMessage CreateSuccessfulClientCredentialTokenResponseM
             string tokenType = "Bearer")
         {
             return CreateSuccessResponseMessage(
-                "{\"token_type\":\"" + tokenType + "\",\"expires_in\":\"" + expiry + "\",\"access_token\":\"" + token + "\"}");
+                "{\"token_type\":\"" + tokenType + "\",\"expires_in\":\"" + expiry + "\",\"access_token\":\"" + token + "\",\"additional_param1\":\"value1\",\"additional_param2\":\"value2\",\"additional_param3\":\"value3\"}");

```

Comment:
Please have a test where one of the param is a JSON array and one of the params is a JSON object

Reply:
Oh right, I will add